### PR TITLE
[42] Source-breaking changes

### PR DESCRIPTION
**Pull request by BasThomas**
Does this belong in the "Commits and PRs" section?

(ref #102)

### CODE REVIEW

File: _drafts/2016-10-13-issue-42.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ author: btb
 
 Doug Gregor [updated](https://github.com/apple/swift-evolution/pull/541) the swift-evolution repository for Swift 4 stage 1. 🎉 Swift 4.0's expected release date is *late 2017*, so mark your calendars!
 
+As Brian Gesiak mentioned in [issue #39](https://swiftweekly.github.io/issue-39/), source-breaking really are [hard to determine](https://github.com/apple/swift/pull/4878#issuecomment-252695257).
```
Comment (BasThomas) [inline _drafts/2016-10-13-issue-42.md]: Can I more easily link to an issue, instead of using the url?

Comment (jessesquires) [inline _drafts/2016-10-13-issue-42.md]: yep 😄 you can just use `/issue-39/`

Comment (jessesquires) [inline _drafts/2016-10-13-issue-42.md]: let's add a bit more here, something like:

> It turns out that Slava's changes to variadic parameters [were breaking](https://bugs.swift.org/browse/SR-2907)

Comment (BasThomas) [inline _drafts/2016-10-13-issue-42.md]: Yeah, much better. thanks! ^_^

**Comment by jessesquires** [comment]
🎉

**Pull request merged**